packages/itwin/tree-widget/src/components/search-bar/SearchBox.tsx

```diff
@@ -38,8 +38,6 @@ export interface SearchBoxProps extends CommonProps {
   resultCount: number;
   /** Callback to currently selected result/entry change */
   onSelectedChanged: (index: number) => void;
-  /** Specify that the <input> element should automatically get focus */
```

Comment:
Is this not a needed prop?

Reply:
this component was only used internally, and we never used the prop. 

also, there was a linter warning re ux, and how we should never set autofocus directly so just removed it altogether

Reply:
okay, sounds good to me.